styles.css

```diff
@@ -2,29 +2,22 @@
   margin: 0;
   padding: 0;
 }
-/* font-family: 'Quicksand', sans-serif;
-COLORS
-light-yellow - #f7e4bf;
-light-blue - #78a7c6;
-dark-blue - #134d71;
-white - #ffffff;
-black - #000000;
-*/
 
 body {
-  background: linear-gradient(0deg, #f7e4bf, #78a7c6 40%, #134d71 85%);
-  text-align: center;
-  font-family: 'Quicksand', sans-serif;
-  color: #ffffff;
   display: flex;
-  justify-content: space-around;
+  text-align: center;
   height: 100vh;
+
+  background: linear-gradient(0deg, #f7e4bf, #78a7c6 40%, #134d71 85%);
+  color: #ffffff;
+
+  font-family: 'Quicksand', sans-serif;
 }
 
 main {
   display: flex;
-  justify-content: space-around;
   flex-direction: column;
+  justify-content: space-around;
   align-items: center;
   width: 100vw;
   margin: 0vh 0vw 18vh 0vw;
```

Comment:
Great job figuring out a way to get your content laid out like you want! I would consider moving this `margin-bottom` to the `.message` selector.

The reason being, I can see a future in which the content on this page would change. In this instance, you might want to take advantage of the full height of `main`. By moving this to `.message`, you're styling the children rather than the container which allows for the container to hold any size/layout of children.